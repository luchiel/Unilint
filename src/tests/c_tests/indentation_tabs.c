int main()
{
	for(int i = 0; i < 5; ++i)
	{
			//bad indent
		printf("%d\n", i);
	}
}