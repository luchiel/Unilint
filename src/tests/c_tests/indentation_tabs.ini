[indentation]
indentation_style = tabs
