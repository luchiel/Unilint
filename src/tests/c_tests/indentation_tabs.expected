5:1: indent too deep
